lib/msal-core/test/UserAgentApplication.spec.ts

```diff
@@ -486,6 +486,14 @@ describe("UserAgentApplication.ts Class", function () {
             expect(authErr.stack).to.include("UserAgentApplication.spec.js");
             done();
         });
+
+        it("throws an error if configured with a null request", () => {
+            try {
+                msal.acquireTokenRedirect();
```

Comment:
I'm not sure this is what you want to do, as this test will pass if it doesn't throw.

Reply:
Checkout https://github.com/domenic/chai-as-promised provides a nice set of chai asserts for tests such as this.

Reply:
good call ... updating!

Reply:
e.g.
```js
return msal.acquireTokenRedirect.should.be.rejectedWith(ClientConfigurationError);
```

Reply:
Could you do this to assert the error details and then add a separate test that does something like:

```typescript
it("throws an error if configured...", () => {
    ...
   expect(msal.acquireTokenRedirect).to.throw(ClientConfigurationError);
});
```
?

Reply:
There is some weird error happening with the iframe wrapping code that would prevent this 
pushing test that works

Reply:
what do you mean hector?

Reply:
not sure I fully understood

Reply:
Basically, one test that checks that the error being thrown *is the right one* (accomplished by your original code) and then another test that checks that *the error is actually being thrown* (the example code I suggested).

Reply:
Although the `correctError = true` hack makes it so you test both at the same time, which is fine (maybe even better).